includes/Support/Path.php

```diff
@@ -1,25 +1,68 @@
 <?php
 namespace App\Support;
 
+use EmptyIterator;
+use Iterator;
+
+/**
+ * Currently there is no notion of "relative" or "absolute"
+ * path as of {@see Path} object, because there is no functionality
+ * of joining paths together. Method {@see Path::toString}
+ * currently returns paths as absolute paths, but {@see Path}
+ * itself can be thought of as both absolute and relative.
+ * The distinction will have to be made when there are methods
+ * in this class that allow joining paths - those methods
+ * will have to decide whether to treat other {@see Path} as
+ * relative or absolute.
+ */
 class Path
```

Comment:
Is this class used anywhere at this moment?

Reply:
Nope. But will be in `FikeSystem`, instead of `string $path`. That will be the next commit in this PR (draft).